migrations/20241127165812-add-expense-attachments-url-indexes.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.addIndex('ExpenseAttachedFiles', ['url']);
+    await queryInterface.addIndex('ExpenseItems', ['url']);
```

Comment:
Maybe only on non-deleted entries? https://github.com/opencollective/opencollective/issues/3205

Reply:
Only ExpenseItems has a `deletedAt` column. I dont know why, but expense attached files does not have one.